packetbeat/_meta/fields.common.yml

```diff
@@ -97,185 +97,40 @@
   description: >
     These fields contain data about the flow itself.
   fields:
-    - name: "start_time"
+    # Not in ECS.
+    - name: event.start
```

Comment:
Interesting. I think APM has similar start timestamps. @graphaelli ?

Reply:
APM has a `span.start` but serves a different a purpose (relative offset from the start of a transaction).  If I understand the usage here correctly, APM would use `@timestamp` as an equivalent.

Reply:
If the two are identical, yes. @andrewkroh I assume in the flow case the two are not identical?

Reply:
In the case of flows we have four unique timestamps.
- `event.start` - time of the first packet
- `event.end` - time of the last packet
- `event.created` - time that the flow emitter (router/switch) generated the record
- `@timestamp` - the time that the flow collector received the event (I could see populating this with one of the other timestamps(